tests/unit/coordinator/plugins/clickhouse/test_parts.py

```diff
@@ -119,7 +118,7 @@ def test_group_files_into_parts_ignores_parts_from_unselected_tables() -> None:
     second_server_files = copy.deepcopy([*TABLE_3_PART_1B])
     tables_replicas = {T1_UUID: DATABASES_REPLICAS, T2_UUID: DATABASES_REPLICAS}
     parts, other_files = group_files_into_parts([first_server_files, second_server_files], tables_replicas)
-    assert parts == []
+    assert not parts
```

Comment:
This, and all the other similar fixes, I assume it's because of the `use-implicit-booleaness-not-comparison` pylint warning, but I think this is a wrong hint and can lead to code that isn't clear about its intent and even bugs:

- you have a test that check that `function_x` returns an empty list
- the linters tell you to booleanize the test
- the `function_x` is modified to return an `list[T] | None` instead of `list[T]`
- now your test is confusing empty list and no list at all, if `function_x` is returning something like "an optional list of allowed things", then the difference is very important.


I'd prefer to disable this lint in .pylintrc.

Reply:
Agree the implicit booleaness leads to ambiguous code. Disabling the warning and reverting these.